scripts/util/create-and-activate-DR.js

```diff
@@ -88,11 +88,33 @@ const createAndActivateDR = async (path, createOnly, activateOnly) => {
   let tx, receipt;
   // Create dispute resolver
   if (!activateOnly) {
+    // create dispute resolver with callers account
+    let adminDisputeResolver = { ...disputeResolver };
+    adminDisputeResolver.admin = adminAddress;
+    adminDisputeResolver.operator = adminAddress;
+    adminDisputeResolver.clerk = adminAddress;
```

Comment:
It would complicate the script right now because you'd have to get the DR admin funded and have that account do the creation, since all the addresses have to be the same as the caller. 

Having the deployer/admin do it is a convenience for now, but when we get to the "opt-in to update" functionality, this script will have to fund all those DR addresses, and we will need the DR admin do the create with all addresses set to it, followed by an upgrade and confirmations from all its proposed new addresses.

So for the moment, I think this is OK, because we'll have to change very soon anyhow.

Reply:
"this script will have to fund all those DR addresses" - The script doesn't have to do any funding. The address owner does.

"ollowed by an upgrade and confirmations from all its proposed new addresses." - The script won't need to do the update. The script only needs to create the DR using the DR admin address (which is the same as the other addresses). Then the protocol ADMIN calls activate. There's no update involved.